Dockerfile

```diff
@@ -2,24 +2,22 @@
 # docker build -t transmission-rss --build-arg UID=1337 --build-arg GID=1337 .
 # docker run -it -v $(pwd)/transmission-rss.conf:/etc/transmission-rss.conf transmission-rss
 
-FROM ruby:alpine
-MAINTAINER henning mueller <mail@nning.io>
+FROM alpine:3 as builder
+RUN apk add gcc libc-dev make ruby-dev
+COPY . /tmp
+WORKDIR /tmp
+RUN \
```

Comment:
No need to combine RUN's in the 'as builder' stage for flatter images.  It will be discarded, and the layers will help debug the build.

Reply:
Good to know! I will change it later.